kuwala/pipelines/osm-poi/src/Downloader.py

```diff
@@ -23,3 +26,38 @@ def start(args):
         file_path += '.osm.pbf'
 
         download_file(url=args.url or file['url'], path=file_path)
+
+    def names_downloader():
```

Comment:
`names_downloader` is a name more suitable for a class than a function. `download_names` seems more fitting. Also please rename the `start` method since we have two functions now. `download_pbf` would be my suggestion.